### ISSUE TITLE
Market Maker on GTK

### ISSUE THREAD

**Issue by ghuznee** [Sifchain/issues2.0#127]

**Comment by ghuznee** [comment]
Reached out to Pecunia.Finance. 

TLDR: The SDK was not updated to allow for it as FE had key priorities in other places overruling anything else by Jon at the time - (Margin, UI v2). Below are the key blockers. Both were with Front End to do. And need to be addressed for Pecunia to pick it up again.
As I was the DevRel PM amongst other things, we figured this out:
"SifchainJS - STATUS: Bugs to be fixed. Stable release is highly recommended
BUGs: Price Discrepancies between the SifchainJS and the UI - 740
Feature Request: Add a missing function on the SifchainJS - 2183
Stakeholders such as Pecunia Finance cannot currently use the SifchainJS on its current version because of the above tickets. Without the tickets being solved, Pecunia Finance cannot go forward with its integration with HummingBot. We can expect other stakeholders to witness similar issues, meaning that the SifchainJS is not ready be used for Community Developer Programs or other initiatives. As an additional consideration, Pecunia Finance is creating this integration being funded by the SifDAO. If they weren’t funded by the SifDAO directly, they would have probably stopped looking at this integration, meaning that the tooling that we have is far from being optimal to be “mass-adopted”

**Comment by ghuznee** [comment]
Bucky comments: It's basically on us to resolve the above

**Comment by ghuznee** [comment]
Here's the Proposal - https://commonwealth.im/sifchain/discussion/3957-onchain-proposal-sifdao-grant-fund-development-of-a-hummingbot-connector-for-sifchain

**Comment by ghuznee** [comment]
Reach out to Petunia about proposition of supporting GTK Margin
